packages/client/src/elements/AppBar/index.tsx

```diff
@@ -107,9 +107,12 @@ const AppBar = ({ tabs = defaultTabs }: AppBarProps) => {
     setActiveTab(defaultActiveTab);
   }, [defaultActiveTab, isMobile]);
 
-  const handleChange = (event: MouseEvent, newValue: number, path: string) => {
+  const handleChange = (event: MouseEvent, newValue: number) => {
     setActiveTab(newValue);
-    history.push(path);
+    const selectedTab = tabs.find((tab) => newValue === (isMobile ? tab.indexMobile : tab.index));
```

Comment:
@Mefjus I just found an issue with that. We handle `indexMobile` as an optional parameter, so on mobile for tabs items which doesn't have `indexMobile` it will always return false.